app/views/posts/show.html.erb

```diff
@@ -9,5 +9,16 @@
   <%= @post.body %>
 </p>
 
+<div id="comments">
+  <% if @post.comments.present? %>
+    <h1>Comments</h1>
+  <% end %>
```

Comment:
This conditional could end _after_ rendering the comments so that if there aren't any comments, the render comments function won't have to run.